overlord/configstate/configcore/refresh.go

```diff
@@ -53,6 +54,15 @@ func reportOrIgnoreInvalidManageRefreshes(tr RunTransaction, optName string) err
 }
 
 func validateRefreshSchedule(tr RunTransaction) error {
+	maxInhibitionDaysStr, err := coreCfg(tr, "refresh.max-inhibition-days")
```

Comment:
Nitpick, if you make the config value a constant you can reuse it in the error message below. Is there a package with all configuration values so that we don't accidentally typo-paste something?

Reply:
It's this package, and none of the entries are consts ATM. It's probably ok to add a task to change that, but IMO it's not within the scope of this PR.

Reply:
I just followed the style for all other config values...